senseicore/src/channels.rs

```diff
@@ -166,11 +166,7 @@ impl ChannelOpener {
             .fee_estimator
             .get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
 
-        // TODO: is this the correct conversion??
-        let sat_per_vb = match fee_sats_per_1000_wu {
-            253 => 1.0,
-            _ => fee_sats_per_1000_wu as f32 / 250.0,
-        } as f32;
+        let sat_per_vb = std::cmp::min(1.0, fee_sats_per_1000_wu as f32 / 250.0);
```

Comment:
I'm still not entirely sure this is right either.  I'm pretty sure 253 "fee_sats_per_1000_wu" is supposed to be 1.0 sat_per_vb.  If you always just divide by 250.0 you get 1.012.  Which I guess probably gets rounded to 1 or I guess even if not it means you'd pay an extra sat for every 100 vbytes?  Not really a big deal.

Matt ended up opening this issue in bdk https://github.com/bitcoindevkit/bdk/issues/608

I'm still not entirely sure the best way to do this for now but maybe your simplification is fine?

Reply:
This is what we do in bitcoin-s so I copied it here:

https://github.com/bitcoin-s/bitcoin-s/blob/master/core/src/main/scala/org/bitcoins/core/wallet/fee/FeeUnit.scala#L230

Reply:
@johncantrell97 @benthecarman given that the minimum fee in sats_per_kwu is 253. Will `std::cmp::min(1.0, fee_sats_per_1000_wu as f32 / 250.0)`  not always produce a fee  rate of `1.0`. Is there something am missing here?